src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,35 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	int max_miners = g_maxMiners.Get();
+	if (max_miners > 0)
+	{
+		gentity_t *tmp = g_entities;
+		int miners = 0;
+		for (int i = 0; i < level.num_entities; i++, tmp++) 
+		{
+			if (tmp->s.eType == entityType_t::ET_BUILDABLE) 
+			{
+				HealthComponent* health = tmp->entity->Get<HealthComponent>();
+				if (tmp->buildableTeam != ent->client->pers.team || !health->Alive())
```

Comment:
Codestyle (hence, not blocking):

inconsistent with next if, and usually the old style files use {} even for one-line blocks, {} being on their own lines. I write *usually* because in this simple function there's already 2 different styles, but the one you're going to include would be a 3rd one (no {}) in this function.

`tmp->buildableTeam != ent->client->pers.team` -> `G_OnSameTeam` is what you want here here, I think.
Also, I don't really like double negations (matter of taste, really): I think `if ( !( G_OnSameTeam( tmp, ent ) && IsAlive( tmpt ) ) ){ continue; }` might be more readable, despite the additional parens.